.scalasteward.conf

```diff
@@ -0,0 +1 @@
+pullRequests.frequency = "0 0 1,15 * ?"
```

Comment:
I'm OK with monthly or bi-weekly if you think that is too infrequent.

Reply:
Maybe let's try bi-weekly. If we get annoyed, we can change it to monthly.